src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -196,6 +209,13 @@ def main():
         )
         _create_file_from_dataframe(_get_attendance_events(section_id), file_path)
 
+    if input_directory is not None:
```

Comment:
To improve readability, what do you think about adding an explanatory variable?

```suggestion
    need_to_process_input_files = input_directory is not None
    if need_to_process_input_files:
```

Reply:
✔️